src/script/client/ClientEntity.ts

```diff
@@ -47,8 +47,7 @@ export class ClientEntity {
     lon?: number;
   };
   meta: {
-    isVerified?: ko.Observable<boolean>;
-    isMLSVerified?: ko.Observable<boolean>;
```

Comment:
This value should not be cached for now. Everytime we need to know if a device is verified or not, we need to ask CoreCrypto. 

later on we might implement some caching, but for now this is too early optimization